prds/watch/MUX-INSERTS-TODO.md

```
@@ -0,0 +1,286 @@
+	•	apps/watch/src/components/VideoCard/MuxVideoCard.stories.tsx
+
+Validation
+	•	New editor can add an insert end-to-end in <15 minutes.
+
+Rollback
+	•	Revert docs and stories.
+
+⸻
+
+Build & CI Guards
+	•	Build validates video-inserts.mux.json against schema.
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)